torch/csrc/jit/ir/alias_analysis.h

```diff
@@ -80,6 +80,9 @@ class AliasDb {
   // Do any nodes write to an alias set inputed/outputed by `n`?
   TORCH_API bool hasWriters(const Node* n) const;
 
+  // Does the node write to any of its inputs
+  TORCH_API bool writeOccursAtNode(Node* n) const;
```

Comment:
Can we compose this out of existing pieces we have, so we can avoid adding another method? I see a prviate `getWrites`, which is very similar, and from the name `isMutable` does the same job as well.

Reply:
yea, i think this was accidentally left in the stack. as you pointed it out it reimplements `writesToAlias`